File: 0137-single-number-ii/0137-single-number-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
  let x ={}
  for(let i=0;i<nums.length;i++){
    let ch = nums[i]
    x[ch]=(x[ch]||0)+1

  }
  let output=0
  for(let key in x){
    if(x[key]===1){
        output =  (Number(key))
    }
  }
  return output
};

File: 0137-single-number-ii/0137-single-number-ii.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        f = {}
        output = []
        for i in range(0,len(nums)):
            ch = nums[i]
            f[ch] = f.get(ch,0)+1
        for key in f:
            if f[key] == 1:
                return key